### PR TITLE
Feat: add TinyImage tool to compress images

### PR DESCRIPTION
**Pull request by chyroc**
@hwchase17

### CODE REVIEW

File: libs/community/langchain_community/tools/tiny_image/tool.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+from typing import Optional
+
+from langchain_core.callbacks import CallbackManagerForToolRun
+from langchain_core.tools import BaseTool
+
+from langchain_community.utilities.tiny_image import TinyImageAPIWrapper
+
+
+class WikipediaQueryRun(BaseTool):
```
Comment (hwchase17) [inline libs/community/langchain_community/tools/tiny_image/tool.py:11]: name

**Review by hwchase17 — COMMENTED** [review]
how do you imagine this being used?

**Comment by chyroc** [comment]
> how do you imagine this being used?

Compress the image and send it to the user as a thumbnail instead of sending the original image to the user. When the user clicks, the original image is displayed.

**Comment by hwchase17** [comment]
tools are largely meant to be called by LLMs, and to me this seems like functionality someone would want to do automatically rather than leave to an LLM to call

**Pull request closed without merging**